src/Appwrite/Resque/Worker.php

```diff
@@ -2,10 +2,17 @@
 
 namespace Appwrite\Resque;
 
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Logger\Log;
+use function get_class;
+
 abstract class Worker
 {
     public array $args = [];
 
+    abstract public function getWorkerName(): string;
```

Comment:
```suggestion
    abstract public function getName(): string;
```

The word `worker` is already obvious from the context. Not need to mention it again.

Reply:
Renamed all `getWorkerName` to `getName` - in abstract class, in every worker and in log where I used this method.